code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -72,14 +76,61 @@
 	if(A.totalStageSpeed() >= 6)
 		power = 2
 
+/datum/symptom/heal/starlight/proc/CanTileHeal(turf/T)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

THIS one should be original_turf.